clang/test/CodeGenSYCL/sycl_unaliased_property.cpp

```diff
@@ -32,48 +32,48 @@ int main() {
     int *b;
     int *c;
     kernel<class kernel_restrict1>([a, b, c]() { c[0] = a[0] + b[0]; });
```

Comment:
Nit - The kernel names are still restrict1 etc.

Reply:
Good eye! They have been renamed.